### ISSUE TITLE
How to Use.

### ISSUE THREAD

**Issue by techzin** [danielkrupinski/VAC-Bypass-Loader#26]
I'm new in the Cheats World,

How to Use this.

Build in Release x86
Close Steam, and Open .exe with admin and Inject Cheat?
Only This?
The Bypass work with every Cheat? Or Only Osiris?

**Comment by hkmodd** [comment]
Correct, for security reasons use Osiris because it gets updated very often
I repeat the steps for being sure with you:
1. Close Steam
2. Compile in Release-x86
3. Run (as administrator ofc) and check for the "initialization successful" prompt
4. Run CS:GO
5. Run your Injector as Administrator too or it won't inject for some reasons

**Comment by Vadenez** [comment]
You can use any cheat, just make sure it's a reliable cheat :).

**Comment by Kr4iNgod** [comment]
Ok, but i need to open vac bypass every time i want to cheat or i just need to open 1 time?
